### ISSUE TITLE
cross platform all apps activate google universe api (google earth api) (.exe request)

### ISSUE THREAD

**Issue by wereshaunsgirls** [andy-portmen/google-shortcuts-cross-platform#6]
I need a cross platform app activation for all google home, 5g, google earth API, sky map, google universe API .exe. This .exe needs to automatically activate all apps or .exe's affiliated with google outer space API (google universe API) to activate it perfectly from outbound auto program procedures (any programming debugs can join auto)

I am developing Google Universe API for google earth (you know how when you enter google earth you see outer space from earth, now you can go to outer space and play a virtual reality video game that is physical, spiritual, and mentally accessible... you can do anything you type into the search field like in google search but it will be an actual word type procedure like shell (SSH) It will be like a wiki dictionary based app but you can have a user avatar, user account, an vehicles, stations, shop stations, businesses and stores factory's, build buildings, houses, basically and second existence on the computer that will be our new internet. 

It will be fully customization. You can create your own peripherals of user video games, (all games will join this universe (street fighter 5, Tekken, Elder Scrolls) Everything will join this system into a any way to play system Google Universe OS (slow to speak))

Businesses will be ran from google universe... You wouldn't have to travel to go to meetings and stuff you can use google universe (Real Reality, Virtual Reality Sync)

Google Universe will be the start up for all OS'es Store businesses etc.

Please understand... It got OKed by Google. They gave me the google earth API to my account.

Basically, You can do ANYTHING POSSIBLE AND EVERYTHING POSSIBLE FROM GOOGLE UNIVERSE (API)

There will be 5 (6) usage fields: Automatic, Beginner, Intermediate, Advanced, Custom, (Hidden)

Please create an app bounce outbound git-hub app log debug .exe link to all associated apps... (used for mental creation of all required apps if some are not fully debugged) then send me a link to wereshaunsgirls@outlook.com and here to my user name by reply.
